sdk/metric/instrument.go

```diff
@@ -77,8 +77,8 @@ type Instrument struct {
 	nonComparable // nolint: unused
 }
 
-// empty returns if all fields of i are their zero-value.
-func (i Instrument) empty() bool {
+// IsEmpty returns if all Instrument fields are their zero-value.
+func (i Instrument) IsEmpty() bool {
 	return i.Name == "" &&
 		i.Description == "" &&
 		i.Kind == zeroInstrumentKind &&
```

Comment:
Why do you have zeroInstrumentKind  vs instrumentKindUndefined?

Reply:
Not sure, guessing that is just a historical artifact. I'll update in its own PR.

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/5433